CHANGELOG.md

```diff
@@ -1,5 +1,94 @@
 # Changelog
 
+## [v0.7.0](https://github.com/delta-io/delta-kernel-rs/tree/v0.7.0/) (2025-02-24)
+
+[Full Changelog](https://github.com/delta-io/delta-kernel-rs/compare/v0.6.1...v0.7.0)
+
+### 🏗️ Breaking changes
+1. Read transforms are now communicated via expressions ([#607], [#612], [#613], [#614]) This includes:
+    - `ScanData` now includes a third tuple field: a row-indexed vector of transforms to apply to the `EngineData`.
+    - Adds a new `scan::state::transform_to_logical` function that encapsulates the boilerplate of applying the transform expression
+    - Removes `scan_action_iter` API and `logical_to_physical` API
+    - Removes `column_mapping_mode` from `GlobalScanState`
+    - ffi: Removes `add_partition_columns` in arrow.c
```

Comment:
nit, this isn't an "ffi" change, it's a change in the example program

Reply:
ah yep good catch thanks!